types/src/app.d.ts

```diff
@@ -0,0 +1,20 @@
+// --- App ---
+// Any types which are shared between different parts of the App
+
+export type Breadcrumb = {
+  name: string;
+  href: string;
```

Comment:
A bit of a nit, but to me `href` indicates a full URL, but here we just link to other parts of the app. Also IMO best practice for routing is to route by name, since that's just internal so we can ensure it never changes, whereas the route slug may change. Therefore I think a type like this makes more sense:

```typescript
export type Breadcrumb = {
  displayName: string; // what is shown in the breadcrumb trail
  routeName: string; // path `name` defined in `app/src/router/index.ts`
};

Reply:
I've changed this to `routeTarget` so that we can feed in params/query args too 👍